<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>616</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QTabWidget" name="EditCategories">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="CreatureEdit">
         <attribute name="title">
          <string>Creatures</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTabWidget" name="CreatureTabs">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="CreatureTemplate">
             <attribute name="title">
              <string>Creature Templates</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout">
                <property name="sizeConstraint">
                 <enum>QLayout::SetMaximumSize</enum>
                </property>
                <item row="0" column="0" rowspan="2" colspan="2">
                 <layout class="QVBoxLayout" name="CreatureTemplateQueryLayout"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="EditCreatureTemplate">
             <attribute name="title">
              <string>Edit Creature Template</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <layout class="QVBoxLayout" name="editCreatureLayout">
                <item>
                 <widget class="QScrollArea" name="editCreatureScrollArea">
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="scrollAreaWidgetContents">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>822</width>
                     <height>423</height>
                    </rect>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <layout class="QVBoxLayout" name="scrollarealayout"/>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="CreatureQuery">
             <attribute name="title">
              <string>Creatures</string>
             </attribute>
             <widget class="QWidget" name="verticalLayoutWidget_4">
              <property name="geometry">
               <rect>
                <x>-1</x>
                <y>-1</y>
                <width>871</width>
                <height>451</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="CreatureQueryLayout"/>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="ItemEdit">
         <attribute name="title">
          <string>Items</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="itemtemplateeditLayout"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Quests</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="DBstatus">
        <property name="text">
         <string>DB Status:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionDatabase_Setup"/>
    <addaction name="actionDBC_Setup"/>
   </widget>
   <widget class="QMenu" name="menuSession">
    <property name="title">
     <string>Session</string>
    </property>
    <addaction name="actionView_Session"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSession"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDatabase_Setup">
   <property name="text">
    <string>Database Setup</string>
   </property>
  </action>
  <action name="actionDBC_Setup">
   <property name="text">
    <string>DBC Setup</string>
   </property>
  </action>
  <action name="actionView_Session">
   <property name="text">
    <string>View Session</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>CreatureTabs</tabstop>
  <tabstop>EditCategories</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
